src/libse/Common/ShotChangeHelper.cs

```diff
@@ -89,6 +88,11 @@ public static void DeleteShotChanges(string videoFileName)
 
         public static double? GetPreviousShotChange(List<double> shotChanges, TimeCode currentTime)
         {
+            if (shotChanges == null || shotChanges.Count == 0)
```

Comment:
To avoid duplication `shotChanges == null || shotChanges.Count == 0` could've been moved to a method 
called `Isvalid` or something similar :)